focus-areas/communal-value/social-listening.md

```diff
@@ -82,8 +82,11 @@ Set up a Data Collection Platform of your choice as described in the “Tools”
 
 To implement the metric any MySQL, smart-sheet, excel, or airtable-like excel datasheet program works fine. This data should be simplified enough to interact with other data interfaces to ensure that data migration is simple, straightforward, and can be automated (such as google data studio). This requires that systems used to implement the Social Listening metric work with CSV and other spreadsheet files, and we heavily recommend open source programs for its implementation.
 
-Once you have this, create a data set with the following data points: <sup>8</sup>
+Once you have this, create a data set with the following data points: [8]
 
+![Data points and description table](images/social-listening_data-points-table.png)
+
+<!--
```

Comment:
I thought we're not supposed to use HTML -- I'd remove the markdown table, now that we have the image for the table.

Reply:
I did a quick test on this, the HTML comments are not recognized hence are not part of the generated PDF.

I just included them so if the working group wanted to change the table image, they wouldn't have to create the whole table again. 

Although, I do think it could cause confusion with members of WG, so it might be simpler if we just decide to remove those comments. Any thoughts?

Reply:
We have different options: 
1. The tables will always be available through the Git Log. 
2. To make them more accessible, we could copy-paste the markdown into an issue for creating the table images. 
3. We could move the table markdown into files in the image folder, with the same name as the image.

The end result should be that there is no hidden content in the metric definition.

Reply:
I think the 1st and 2nd options are the simplest but the WG can decide whichever way suits them.

Removing the commented tables.